Classes/Messages.php

```diff
@@ -0,0 +1,22 @@
+<?php
+class Messages{
+    public static function setMsg($text, $type){
```

Comment:
If you're using PHP 7 always strict the passing parameter type and return type:
```suggestion
    public static function setMsg(string $text, string $type): void {
```